NOTICE

```diff
@@ -0,0 +1,15 @@
+   Red Hat 3scale API Management Platform Porta
```

Comment:
I didn't know if I should write `Porta` or `System` 🤷‍♀️

Reply:
I am not even sure we say "Platform" anymore

Reply:
I would just avoid using Platform just to not being confused.

Reply:
ok, so should I push with `Red Hat 3scale API Management User Interface` instead? @hallelujah 
It is what we currently have in the description of the repository:
![image](https://user-images.githubusercontent.com/11318903/49375400-746c8900-f704-11e8-9967-f87b320a97f7.png)

Reply:
I really do not know. But yes. Something along:

"Red Hat 3scale API Management console, Porta"
@thomasmaas